operations.cc

```diff
@@ -0,0 +1,92 @@
+#include <assert.h>
+#include "operations.h"
+#include "ioperandsmemory.h"
+
+namespace PolishCalcComponent {
+
+value_t add(IOperandsMemory* memory){
+    auto b = memory->getOperand();
+    auto a = memory->getOperand();
+
+    memory->putOperand(a + b);
+}
+
+value_t sub(IOperandsMemory* memory){
+    auto b = memory->getOperand();
+    auto a = memory->getOperand();
+
+    memory->putOperand(a - b);
+}
+
+value_t div(IOperandsMemory* memory){
+    auto b = memory->getOperand();
+    auto a = memory->getOperand();
+
+    memory->putOperand(a / b);
+}
+
+value_t mul(IOperandsMemory* memory){
```

Comment:
Looks like all these functions uses only IOperandsMemory data. I would say that it should be part of IOperandsMemory responsibility